src/components/views/settings/tabs/user/SecurityUserSettingsTab.js

```diff
@@ -243,7 +244,7 @@ export default class SecurityUserSettingsTab extends React.Component {
                         <label>{_t("Session key:")}</label>
                         <span><code><b>{identityKey}</b></code></span>
                     </li>
-                </ul>
+                </ul>}
```

Comment:
Please do our changes on new lines whenever possible to minimise merge conflicts.

Reply:
I have been able to move it to new line. So, I have taken action for this item.